extensions/native/recursion/src/halo2/wrapper.rs

```diff
@@ -197,3 +206,45 @@ fn emit_wrapper_circuit_metrics(agg_circuit: &AggregationCircuit) {
     let total_cell = total_advices + total_lookups + stats.gate.total_fixed;
     metrics::gauge!("halo2_total_cells").set(total_cell as f64);
 }
+
+fn gen_evm_verifier_sol_code(
```

Comment:
comment and link that this is copied from `gen_evm_verifier` in `snark-verifier` and the only change is to return the sol code instead of writing to file.

We may also consider just moving this change into `snark-verifier` itself?

Reply:
on second thought, let's move it into `snark-verifier`